bsdlib/doc/CHANGELOG.rst

```diff
@@ -5,6 +5,18 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.6.2
+************
+
+* TLS session cache is again disabled by default due to missing support in modem firmware versions older than 1.1.1 included
+* :cpp:func:`sendto()` will now set errno to ``NRF_EISCONN`` instead ``NRF_EINVAL`` when passing an address if the socket type is ``NRF_SOCK_STREAM``
+* calling :cpp:func:`connect()` on a already connected socket will now properly return ``NRF_EISCONN`` instead of ``NRF_EBADF``
+* sockets with family ``NRF_AF_LTE`` must now be created with type ``NRF_SOCK_DGRAM``
```

Comment:
Maybe this can be changed to proper validation of family, type, and protocol arguments to socket(). Should probably also mention that 0 is not accepted to any of the arguments as 0 is in some cases accepted on posix/bsd sockets to create default socket configuration. And was commonly used to create AT sockets before.

Reply:
Implicitly, 0 is not accepted if we say type MUST be DGRAM, so I don't change that. I can also add: 
- Improved validation of family, type, and protocol arguments in socket()